common/djangoapps/course_modes/models.py

```diff
@@ -70,6 +71,7 @@ class CourseMode(models.Model):
 
     # Modes that allow a student to pursue a verified certificate
     VERIFIED_MODES = ["verified", "professional"]
+    NO_ID_PROFESSIONAL_MODES = ["no-id-professional"]
```

Comment:
There is only one "no-ID professional" mode. As such, this doesn't need to be a list. If you want to use a list, I'd suggest changing the constant's name to `NON_VERIFIED_MODES`, making it the counterpart of `VERIFIED_MODES` (which seems to be what you were going for).